tests/test_variable_manipulation.py

```diff
@@ -68,42 +68,89 @@ def test_find_or_check_numerical_variables(df_vartypes, df_numeric_columns):
     assert _find_or_check_numerical_variables(df_numeric_columns, 2) == [2]
 
 
-def test_find_or_check_categorical_variables(df_vartypes, df_numeric_columns):
+def test_find_or_check_categorical_variables(
+    df_vartypes, df_datetime, df_numeric_columns
+):
     vars_cat = ["Name", "City"]
     vars_mix = ["Age", "Marks", "Name"]
 
-    assert _find_or_check_categorical_variables(df_vartypes, vars_cat) == vars_cat
-    assert _find_or_check_categorical_variables(df_vartypes, None) == vars_cat
-
+    # errors when vars entered by user are not categorical
     with pytest.raises(TypeError):
         assert _find_or_check_categorical_variables(df_vartypes, "Marks")
-
+    with pytest.raises(TypeError):
+        assert _find_or_check_categorical_variables(df_datetime, "datetime_range")
+    with pytest.raises(TypeError):
```

Comment:
to be absolutely pedantic, could we add a test passing a single variable in a list?

for example add `assert _find_or_check_categorical_variables(df_datetime,[ "datetime_range"])`

this should fail as well, right?